pymc3/variational/callbacks.py

```diff
@@ -0,0 +1,39 @@
+import scipy.stats as stats
+import numpy as np
+
+
+class Callback(object):
+    def __call__(self, approx, loss, i):
+        raise NotImplementedError
+
+
+class CheckLossConvergence(Callback):
```

Comment:
Very elegant.

We should also add the classic ADVI convergence criterion: https://github.com/pymc-devs/pymc3/blob/master/pymc3/variational/advi.py#L180